src/reflow/mod.rs

```diff
@@ -44,13 +42,18 @@ impl Checker for Reflow {
 fn reflow_inner<'s>(
```

Comment:
Without comments nobody will remember what reference `range` and `unbreakable_ranges` have